test/e2e/proxy_test.go

```diff
@@ -146,6 +146,14 @@ func probeClientIPFromPod(data *TestData, pod string, baseUrl string) (string, e
 	return host, err
 }
 
+func reverseStrs(strs []string) []string {
```

Comment:
Out of curiosity, why it needs to reverse the urls?

Reply:
This is a little confused to read.

 For example, there are two Nodes, and assumed that A is 192.168.1.1; B is 192.168.1.2.
Assumed that the port number of NodePort is 3000. The test URLs are generated by be following code:
```
nodeIPs := []string{controlPlaneNodeIPv4(), workerNodeIPv4(1)}
...
var urls []string
for _, nodeIP := range nodeIPs {
    urls = append(urls, net.JoinHostPort(nodeIP, portStr))
}
```
To make Node A connect to Node B's NodePort and Node B connects to Node A's NodePort in a `for` loop, the slice should be reversed.

Reply:
Thanks for the explanation.